Makefile

```diff
@@ -1,10 +1,14 @@
 FLASK_PORT=5050
 
+ifndef FLASK_HOST
```

Comment:
Huh, in all the years I've been working with C++ I never came across Makefile conditionals. That's interesting to know.